_data/reference.yml

```diff
@@ -1,53 +1,60 @@
 bigheader: "Reference Documentation"
 abstract: "Design docs, concept definitions, and references for APIs and CLIs."
 toc:
-- docs/reference.md
+- docs/reference/index.md
 
-- title: "Kubernetes Resource Types"
+- title: API Reference
   section:
-  - title: Version 1.6
-    path: /docs/resources-reference/v1.6/
-  - title: Version 1.5
-    path: /docs/resources-reference/v1.5/
-
-- title: "Kubernetes API"
-  section:
-  - title: Overview
-    path: /docs/concepts/overview/kubernetes-api/
-  - title: Version 1.6
+  - title: v1.6
     path: /docs/api-reference/v1.6/
-  - title: Version 1.5
+  - title: v1.5
     path: /docs/api-reference/v1.5/
+  - docs/reference/extensions/v1beta1/definitions.html
```

Comment:
I don't think this page is necessary. Also it's not a release version like v1.6.

The huge v1.6 API reference page covers everything this /v1beta1/ page has. cc:@pwittrock to confirm